legend-engine-xt-relationalStore-spanner-execution/src/main/resources/META-INF/services/org.finos.legend.engine.plan.execution.stores.relational.connection.ConnectionExtension

```diff
@@ -0,0 +1 @@
+org.finos.legend.engine.plan.execution.stores.relational.connection.spanner.extensions.SpannerConnectionExtension
```

Comment:
we need to create a file for RelationalConnectionExtension too with the same content, as SpannerConnectionExtension implements both RelationalConnectionExtension and ConnectionExtension. See [here](https://github.com/finos/legend-engine/tree/master/legend-engine-xt-relationalStore-sqlserver-execution/src/main/resources/META-INF/services)